modules/Data/Data.yaml

```diff
@@ -91,6 +91,11 @@ enums:
       electrophysiology:
         description: Data generated from an electrophysiology assay.
         meaning: http://purl.obolibrary.org/obo/ERO_0000564
```

Comment:
I am getting a "Term not found" error using the link :http://purl.obolibrary.org/obo/ERO_0000564 

Since there are multiple electrophysiology assays, it may be time to update this to be more granular.

Reply:
Since this term is out of scope for this PR, could you add this as an issue? BTW, this is for the dataType module, not the assay module - so maybe it's fine to have the broader term here?